env0/templates.go

```diff
@@ -208,6 +208,12 @@ func getTemplateSchema(templateType TemplateType) map[string]*schema.Schema {
 			Description: "the cloudformation file name. Required if the template type is cloudformation",
 			Optional:    true,
 		},
+		"is_terragrunt_run_all": {
```

Comment:
This is all that's required. The readResourceData and writeResourceData do all the work automatically.